Value_converter/main.py

```diff
@@ -1,59 +1,54 @@
 import tkinter as tk
 from icecream import ic
 
+from http_request import get_response
+
 #inizializza finestra
 window = tk.Tk()
 window.title("Converter")
 window.geometry("500x200")
 
-euro_value = 1
-dollar_value = 2
-yen_value = 3
-pound_value = 4
 
-currencies = {
-    "euro": euro_value,
-    "dollar": dollar_value,
-    "yen": yen_value,
-    "pound": pound_value
-}
+currencies = []
+
+ic(get_response)
 
+#crea entry form
 entry = tk.Entry(window)
 entry.pack()
 
+#converti il valore inserito con la valuta scelta
 def Converter(currency_value,input_value):
         input_value = str(input_value.get())
         
-        float_value = is_float(input_value)
+        float_value = is_float(input_value) #check se il valore è float
 
         if float_value: 
                 input_value = float(input_value)
                 ic(input_value)
-                new_value = input_value * input_value/currency_value
+                new_value = input_value * currency_value #formula di conversione
                 return new_value 
         else:
             l_result.config(text="Non hai inserito un valore valido da convertire! \n Prova ad inserire un numero o cambiare la virgola con un punto")
 
-
+#verifica se il valore è float
 def is_float(value):
```

Comment:
Potrebbe essere una buona idea avere delle funzioni utility come questa e/o `show_result` in un modulo a parte